src/Platform/Microsoft.Testing.Extensions.VSTestBridge/ObjectModel/FastFilter.cs

```diff
@@ -195,14 +195,15 @@ private void AddProperty(string name, string value)
         {
             if (!_filterDictionaryBuilder.TryGetValue(name, out ImmutableHashSet<string>.Builder? values))
             {
-                values = ImmutableHashSet.CreateBuilder<string>(StringComparer.OrdinalIgnoreCase);
+                values = ImmutableHashSet.CreateBuilder(StringComparer.OrdinalIgnoreCase)!;
```

Comment:
Why is the suppression needed?

Reply:
It's complaining about inconsistent nullability.

Reply:
Oh, was the code previously creating `ImmutableHashSet<string>` and it's not `ImmutableHashSet<string?>` ? In that case, I think it's better to keep the type parameter explicitly specified as `string`